### DISCUSSION TITLE
Ring MQTT over the LAN network.. is possible?

### DISCUSSION THREAD

**Discussion by vasco80hassio** [tsightler/ring-mqtt#622] [General Discussion]
Hi all... sorry for the stupid question, but I'm not very experienced.

Does the implementation of ring mqtt allow to send mqtt messages from ring alarm device (Casa Base Station) locally, without an internet connection avaible?
I implemented ring-mqtt and everything works, but the broker only receives messages if the internet connection is online

Is it possible to configure the broker's address and user on the Casa Base Station, so that it can communicate with the broker over the LAN network, without going on the internet?

Thanks

**Comment by tsightler** [comment]
No, it is not possible.  Please [read the FAQ](https://github.com/tsightler/ring-mqtt/wiki/Frequently-Asked-Questions-(FAQ)), as the 3rd question specifically addresses this.

**Reply by vasco80hassio** [reply]
Thanks... I'm sorry, but I had searched on the topics and on the wiki page, but didn't sawa the faq page and that question.
